docs/_source/getting_started/installation/configurations/elasticsearch.md

```diff
@@ -29,19 +26,8 @@ http(s)://user:passwd@elastichost
 
 Below you can see a screenshot for setting up a new *argilla* Role and its permissions:
 
-![Argilla Role and permissions in ES](https://user-images.githubusercontent.com/2518789/142883104-f4f20cf0-34a0-47ff-8ee3-ab9f4644271c.png)
-
-
-## Change index analyzers
-
-By default, for indexing text fields, Argilla uses the `standard` analyzer for general search and the `whitespace`
-analyzer for more exact queries (required by certain rules in the weak supervision module). If those analyzers
-don't fit your use case, you can change them using the following environment variables:
-`ARGILLA_DEFAULT_ES_SEARCH_ANALYZER` and `ARGILLA_EXACT_ES_SEARCH_ANALYZER`.
```

Comment:
Do we no longer support specifying analyzer or is this excess information?

Reply:
This is something that we do not support for feedback datasets. I would like to move the feedback dataset features supported as the default ones.

Reply:
Make sense then.